samigo/samigo-hibernate/src/java/org/sakaiproject/tool/assessment/data/dao/assessment/ItemTag.java

```diff
@@ -28,6 +28,7 @@
 
 /**
  * A binding between an {@link ItemData} and a {@code TagService}-managed {@code Tag}
+ * TODO: Should extend TagIfc
```

Comment:
Could you expand a bit on this?

Reply:
I added TagIfc and it can be the parent of ItemTagIfc. I noticed now that i placed the comment into the wrong file. When ItemTagIfc extends TagIfc, a few lines can be deleted, but I wanted to omit this changes from this PR, as it's not adding much value here.

Reply:
I moved the comment to the ItemTagIfc file now.